src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,113 @@
+import { useEffect, useState } from 'react';
+import { Loader } from './Loader';
+import { PersonLink } from './PersonLink';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+export const PeopleTable = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState('');
+
+  const { slug } = useParams();
+
+  const isSelected = people.find(person => person.slug === slug);
+
+  useEffect(() => {
+    setLoading(true);
+    getPeople()
+      .then(setPeople)
+      .catch(() => {
+        setError('Unable to load data');
```

Comment:
The error state is set when the data fetching fails, but it is not cleared when a new fetch attempt is successful. Consider resetting the error state at the beginning of the `useEffect` or when a new fetch attempt is made to ensure the error message does not persist incorrectly.